barao/comida.tex

```diff
@@ -534,7 +534,10 @@ \subsection{Bares, lanchonetes e restaurantes}
         \\Endereço: R. José Martins, 751
         \\Site: \url{namastesalad.com.br}
         \\Telefone: (19) 3289-4178
-
+	
+	\item	\textbf{De La Rua:} Food Truck especializado em burritos.
+		\\Endereço: Av. Romeu Tortima, em frente ao Wizard.
+    
```

Comment:
Quase, só tira os trailing whitespace. (espaços inúteis nas linhas vazias acima e abaixo do item, deixa elas realmente vazias).